README.md

```diff
@@ -157,13 +157,12 @@ toolbox/nfd/wait_gpu_nodes.sh
 Cluster
 -------
 
-- [x] Add a GPU node on AWS
+- [x] Set number of nodes with given instance type on AWS
 ```
-./toolbox/cluster/scaleup.sh
-```
-   - [x] Specify a machine type in the command-line, and skip scale-up if a node with the given machine-type is already present
-```
-./toolbox/cluster/scaleup.sh <machine-type>
+./toolbox/cluster/set_scale.sh <machine-type> <replicas>
+
+Example usage:
+./toolbox/cluster/set_scale.sh g4dn.xlarge 2
```

Comment:
maybe you can add `[--force]` here, and a line of comment about it

we'll move that to the documentation soon anyway